server/models/CalculationSummaryViewModel.ts

```diff
@@ -90,4 +90,12 @@ export default class CalculationSummaryViewModel {
   private allSentencesSupported(): boolean {
     return !this.sentencesAndOffences.find(sentence => !SentenceTypes.isSentenceSds(sentence))
   }
+
+  public isErsedElligible(): boolean {
+    return !!this.sentencesAndOffences?.find(sentence => SentenceTypes.isSentenceErsedElligible(sentence))
```

Comment:
I think here you can use `.some` in place on `.find` - hence not not needing to use the `!!`
Similar thing else where

Reply:
Done.